commit e1d5ea269ab971578159073ce64f491f19c9cd8f (refs/changes/84/3984/2)
Author: Jan Wyszynski <jan.wyszynski@joyent.com>
Date:   2018-05-22T23:35:40+00:00 (1 year, 5 months ago)
    
    MORAY-472 moray handler-error dtrace probe is not useful
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/lib/control.js b/lib/control.js
index 3f6cd02..f01adb5 100644
--- a/lib/control.js
+++ b/lib/control.js
@@ -161,7 +161,7 @@ function handlerPipeline(options) {
                 if (err) {
                     dtrace.fire('handler-error', function () {
                         return ([req.msgid, req.req_id, route, handler.name,
-                            err.toString]);
+                            err.toString()]);
                     });
                 }
                 dtrace.fire('handler-done', function () {
